builder/fetch.sh

```diff
@@ -8,3 +8,5 @@ go mod download "$goPackagePath@$version"
 dir=$(go mod download --json "$goPackagePath@$version" | jq -r .Dir)
 
 cp -r $dir $out
+
+find $out -iname ".ds_store" | xargs -r rm -r
```

Comment:
it should be alright here since it downloads from network into a temporary directory.

Reply:
I think so too, but on the off chance that some sources happen to contain a `.ds_store` I want to make sure that we don't have any difference in behaviour between the Go side of things and the Nix side of things.

Reply:
i see 👍